### ISSUE TITLE
Single crop validation accuracy

### ISSUE THREAD

**Issue by zixeljupo** [MahdiHajibabaei/unified-embedding#5]
Hi,
I'm a bachelor student, and I'm going to use and cite your work in my bachelors thesis.
Could you please share single crop accuracy for ResNet-20 trained with simple softmax loss (which is later fine-tuned with Large Margin softmax loss)? This accuracy should be stated in log file like existing log file for Large Margin softmax. (https://github.com/MahdiHajibabaei/unified-embedding/blob/master/logs/LM/LM_Ident_512D.log)

**Comment by MahdiHajibabaei** [comment]
Hi,

I'm happy that you found interest in this work. I added the log file to the https://github.com/MahdiHajibabaei/unified-embedding/blob/master/logs/LM/ResNet-20_512D.log . The single crop validation accuracy reaches top-1: 80.10 % and top-5: 92.56 %.

**Comment by zixeljupo** [comment]
Thank you very much! It is also very impressive that one could achieve 95% accuracy for 1250 classes. I'm closing this issue.
